epsagon/__init__.py

```diff
@@ -28,7 +28,7 @@ def dummy_wrapper(func):
     step_lambda_wrapper = dummy_wrapper  # pylint: disable=C0103
     azure_wrapper = dummy_wrapper  # pylint: disable=C0103
     python_wrapper = dummy_wrapper  # pylint: disable=C0103
-    EpsagonFlask = dummy_wrapper  # pylint: disable=C0103
```

Comment:
is no one using this? you are gonna break someones code (like ours)

Reply:
The current `EpsagonFlask` breaks the convention actually. Users expects to import `flask_wrapper`